README.md

```diff
@@ -56,6 +56,14 @@ pip install --upgrade -r tests/requirements.txt
 make
 ```
 
+Licensing
+----------
+
+PyAV python package is available under [BSD3 license](LICENSE.txt).
+[PyPI][pypi] and [conda-forge][conda-forge] binaries are shipped with FFmpeg
```

Comment:
We are not in charge of the conda-forge binaries, I really have no idea what they link FFmprg against (especially as they seem to use third party binaries for Windows), so I'd rather keep that out of our license files.

Reply:
Makes sense - didn't know that, sorry.

Reply:
No worries, we do mention the conda installation method in our docs, we should maybe make it clearer this is a distribution outside our control.